aztec/src/main/kotlin/org/wordpress/aztec/AztecParser.kt

```diff
@@ -643,6 +585,19 @@ class AztecParser {
             }
             i++
         }
+
+        if (nl == 0 && text.length > i && text[i] == '\n') {
+            consumeCursorIfThere(out, text, i)
+        }
+    }
+
+    private fun consumeCursorIfThere(out: StringBuilder, text: CharSequence, position: Int) {
```

Comment:
The name `consumeCursorIfThere` is a little ambiguous.  Adding comments describing the function and/or renaming the `There` part of the function would be helpful.